File: src/main/java/br/com/jiratorio/repository/LeadTimeRepository.java
package br.com.jiratorio.repository;

import br.com.jiratorio.domain.LeadTime;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface LeadTimeRepository extends CrudRepository<LeadTime, Long> {

    void deleteByIssueId(Long id);

}

File: src/main/java/br/com/jiratorio/repository/LeadTimeRepository.kt
package br.com.jiratorio.repository

import br.com.jiratorio.domain.entity.LeadTime
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface LeadTimeRepository : CrudRepository<LeadTime, Long> {

    fun deleteByIssueId(id: Long?)

}
